GVRf/Sample/gvrtextviewsample/assets/gvr_note4.xml

```diff
@@ -14,9 +14,39 @@
  * limitations under the License.
  */-->
  
-<lens name="N4">
-    <distortion unit="0.02825" k0="1.0" k1="0.125" k2="0.125" k3="0.0" disparity="-0.008" distortion-scale="1.0"/>
```

Comment:
If those were unused ones in the current context, I think it should be okay to clean them up.